week-5/DaInglichTim/queries/README.md

```diff
@@ -0,0 +1,77 @@
+# Solutionary
+
+## Show the total amount of sales in January 2009
+
+```sql
+SELECT COUNT(id_transaction)
+FROM transactions;
+```
+
+## Build a report of number of sales and total amount by each B2B customer
+
+```sql
+SELECT t.sold_by, COUNT(*), SUM(p.price)
+FROM transactions as t
+  INNER JOIN products as p on t.id_product = p.id_product
+GROUP BY t.sold_by;
+```
+
+## What is your best product?
+
+```sql
+SELECT p.product, COUNT(*) as counter
+FROM transactions as t
+  INNER JOIN products as p on t.id_product = p.id_product
+GROUP BY p.product
+ORDER BY counter DESC LIMIT 1;
+```
+
+## What's the preferred payment type?
+
+```sql
+SELECT payment_type, COUNT(*) as counter
+FROM transactions
+GROUP BY payment_type
+ORDER BY counter DESC LIMIT 1;
+```
+
+## Per each tour, where do customers come from? Show the top 5 locations
```

Comment:
Each query is basically the same but only changes the `p.product` value, do you think this could have been improved to do it in single query and get the data for each possible tour name?